tests/torch/data/reference_graphs/const_sparsity/inception_v3.dot

```diff
@@ -311,129 +311,118 @@ strict digraph  {
 "309 Inception3/InceptionC[Mixed_6e]/BasicConv2d[branch_pool]/NNCFBatchNorm2d[bn]/batch_norm_0" [id=309, type=batch_norm];
 "310 Inception3/InceptionC[Mixed_6e]/BasicConv2d[branch_pool]/relu__0" [id=310, type=relu_];
 "311 Inception3/InceptionC[Mixed_6e]/cat_0" [id=311, type=cat];
-"312 Inception3/InceptionAux[AuxLogits]/avg_pool2d_0" [id=312, type=avg_pool2d];
```

Comment:
In Inceptionv3, the aux branch is gone since we are now building graphs for the eval mode.